recipe/meta.yaml

```diff
@@ -31,7 +31,7 @@ requirements:
     - jinja2 >=3.1.3,<4
     - click >=7.0,<9.0
     - python-dateutil >=2.0,<3
-    - importlib-metadata >=6.0,<7
+    - importlib_metadata >=6.0,<7
```

Comment:
I would rely on an arch-specific variant `importlib-metadata` **not noarch** `importlib_metadata`